website/tailwind.config.js

```diff
@@ -71,10 +71,10 @@ module.exports = {
             "--tw-format-quotes": firezoneColors["night-rider"][700],
             // "--tw-format-quote-borders": theme("colors.orange[300]"),
             // "--tw-format-captions": theme("colors.orange[700]"),
-            "--tw-format-code": firezoneColors["electric-violet"][800],
-            "--tw-format-code-bg": firezoneColors["electric-violet"][50],
-            "--tw-format-pre-code": firezoneColors["electric-violet"][800],
-            "--tw-format-pre-bg": firezoneColors["electric-violet"][50],
+            // "--tw-format-code": firezoneColors["electric-violet"][800],
+            // "--tw-format-code-bg": firezoneColors["electric-violet"][50],
+            // "--tw-format-pre-code": firezoneColors["electric-violet"][800],
+            // "--tw-format-pre-bg": firezoneColors["electric-violet"][50],
             // "--tw-format-th-borders": theme("colors.orange[300]"),
             // "--tw-format-td-borders": theme("colors.orange[200]"),
             // "--tw-format-th-bg": theme("colors.orange[50]"),
```

Comment:
What's the reason to not just delete these and let source control keep the history?

Reply:
Hm, we could -- it's just a little easier to have them here to see all of the color options available to the blog and docs styles, otherwise it might require digging around a bit more.